config/stack_ufs_s2s_model.yaml

```diff
@@ -0,0 +1,251 @@
+cmake:
```

Comment:
I can check, but how different is this from `stack_noaa.yaml`

Reply:
The only differences are inclusion of the jpeg compile and the inclusion of szip functionality for the hdf5 compile.  I can remove them and see how the compile goes.  If it still works, I'll simply remove that config.

Reply:
jpeg is ON on stack_noaa.yaml.
The UFS uses zlib for compression and that is why the szip is OFF.